hsDarkIde.css

```diff
@@ -77,6 +77,44 @@ body.design-manager.ext-dark-theme .HorizontalScrollContainer__ScrollBar-cSRRR{
     bottom: 0.3rem;
     background-color: #3E8CBC;
 }
+
+
+/* ******* 
+* Horizontal Tab Scrollbar -- Height Modification 
+* Kind of a convoluted height modification to the base elements surrounding the scroll.
+* Also looks like some injected styles might be impacting this. 
+* Possibly will need to address this post-hoc.
+**********/
+html.ext-ui-tweaks .HorizontalScrollContainer__ScrollBar-cSRRR {
+    height: 10px; /* increased from something like 3px */
+    bottom: 10px;
+    border-radius: 7px;
+    cursor: move; /* cursor state for easier idendification of active state */
```

Comment:
I dig the change of cursor to indicate hovering it

wonder if ew-resize might be a better cursor to use though.

Not sure honestly. might take that up in slack to see what people think.

Otherwise I think all of these tweaks are good!